### PR TITLE
Fix cmd httpd

### PR DESCRIPTION
**Pull request by dylanwh**

### CODE REVIEW

File: scripts/entrypoint.pl [hunk 1]
```diff
@@ -38,7 +38,8 @@
 sub cmd_httpd  {
     check_data_dir();
     wait_for_db();
-
+    run( 'perl', 'checksetup.pl', '--no-template' );
+    run( '/usr/sbin/httpd', '-DFOREGROUND', '-f', '/app/httpd/httpd.conf' );
```
Comment (floatingatoll) [inline scripts/entrypoint.pl]: Starting Apache seems sensible enough.

File: scripts/entrypoint.pl [hunk 2]
```diff
@@ -249,7 +250,7 @@ sub wait_for_db {
         $dbh = DBI->connect(
             $dsn,
             $c->{db_user},
-            $c->{db_pass}, 
+            $c->{db_pass},
```
Comment (floatingatoll) [inline scripts/entrypoint.pl]: Whitespace nit without any mention in a commit message.

File: scripts/entrypoint.pl [hunk 3]
```diff
@@ -38,7 +38,7 @@
 sub cmd_httpd  {
     check_data_dir();
     wait_for_db();
-
+    run( '/usr/sbin/httpd', '-DFOREGROUND', '-f', '/app/httpd/httpd.conf' );
```
Comment (floatingatoll) [inline scripts/entrypoint.pl]: As before, httpd seems fine. I guess checksetup isn't needed?

Comment (dylanwh) [reply scripts/entrypoint.pl]: Yep, looks like it

**Pull request merged**